tools/test_modules/m11500.pm

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env perl
+
+##
+## Author......: See docs/credits.txt
+## License.....: MIT
+##
+
+use strict;
+use warnings;
+
+use Digest::CRC qw (crc32);
+
+sub module_constraints { [[0, 255], [8, 8], [0, 31], [8, 8], [-1, -1]] }
+
+sub module_generate_hash
+{
+  my $word = shift;
+  my $salt = shift;
+
+  my $digest = crc32 ($word);
```

Comment:
@jsteube I'm not sure if I should concat `$salt` here or not.

Reply:
The perl module does not work as expected, therefore we just set it to fixed zero. You can remove it and use sprintf ("%08x:00000000", $digest);